.circleci/Dockerfile

```diff
@@ -11,40 +11,40 @@ FROM    circleci/golang:1.9
 
 # Also installed docker, docker-compose, dockerize, jq
 
-ARG     K8S_VER=v1.7.4
+ARG     K8S_VER=v1.9.2
+ARG     ETCD_VER=v3.2.15
+ARG     MINIKUBE_VER=v0.25.0
+ARG     HELM_VER=v2.8.0
```

Comment:
Recommend changing this to 2.7.2 as our documentation indicates Helm >= 2.7.2 as a minimum pin.  In addition, Helm 2.8.0 is broken: https://github.com/istio/istio/issues/3176